.gitignore

```diff
@@ -0,0 +1 @@
+/.idea/
```

Comment:
normally better to add a global gitignore then you don't need to do this to all projects that you touch :grin: